deployment/README.md

```diff
@@ -134,3 +134,98 @@ Run the following commands to apply the **container_file_t** SELinux label to th
 # semanage fcontext -a -t container_file_t "<STORAGE-PATH>(/.*)?"
 # restorecon -Rv <STORAGE-PATH>
 ```
+Add a new section to https://github.com/IBM/Db2/tree/develop/deployment.
```

Comment:
Did you intend to have this line?

Reply:
Good catch. No, it's not intended. I'll remove this line.